packages/nexrender-worker/src/index.js

```diff
@@ -72,12 +72,26 @@ const start = async (host, secret, settings) => {
     const client = createClient({ host, secret });
 
     do {
-        let job = await nextJob(client, settings); {
-            job.state = 'started';
-            job.startedAt = new Date()
+        let job = await nextJob(client, settings);
+
+        if (job.jobId && job.logger) {
```

Comment:
As re: the comment on the other PR, we should add the credentials to the `ct` object, so this may need to change.